nntrainer/dataset/random_data_producers.cpp

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   random_data_producers.cpp
+ * @date   09 July 2021
+ * @brief  This file contains various random data producers
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+#include <random_data_producers.h>
+
+#include <base_properties.h>
+#include <node_exporter.h>
+#include <util_func.h>
+
+namespace nntrainer {
+
+/**
+ * @brief Props containing min value
+ *
+ */
+class PropsMin : public Property<float> {
```

Comment:
this is minor but how about PropsRangeMin? corresponding key would become range_min? or you could also consider range_start.

Reply:
This will affect upcoming PR which uses this property so I'll do it later :)

Reply:
There are some tests related to the property here, I'll make it as a separate PR later :)

Reply:
alight.